src/Workleap.DomainEventPropagation.Tests/Subscription/AzureSystemEvents/AzureSystemEventGridWebhookHandlerTests.cs

```diff
@@ -42,7 +43,7 @@ public async Task GivenAzureSystemEventIsFired_WhenTopicIsNotSubscribedTo_ThenAz
 
         await azureSystemEventGridWebhookHandler.HandleEventGridWebhookEventAsync(eventGridEvent, systemEventData, CancellationToken.None);
 
-        azureSystemEventHandler.Verify(x => x.HandleAzureSystemEventAsync(It.IsAny<MediaJobFinishedEventData>(), CancellationToken.None), Times.Never);
+        A.CallTo(() => azureSystemEventHandler.HandleAzureSystemEventAsync(A<MediaJobFinishedEventData>._, A<CancellationToken>._)).MustNotHaveHappened();
     }
 
     [Fact(Skip = "This is failing, we'll investigate later")]
```

Comment:
Is this test still failing after you've switched to `FakeItEasy` or not?

Reply:
Yeah I tried it and it still fails, didn't look further into it tho